packages/unit-vitest/src/templates/base/vitest.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from 'vitest/config';
+import vue from '@vitejs/plugin-vue';
+import { quasar, transformAssetUrls } from '@quasar/vite-plugin';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  test: {
+    environment: 'happy-dom',
+    setupFiles: 'test/vitest/setup-file.ts',
+    include: [
```

Comment:
I added this to avoid Vitest to pick up Cypress test files, it follows the same conventions we defined for Jest AE some time ago